mappings/net/minecraft/structure/EndCityGenerator.mapping

```diff
@@ -39,6 +39,9 @@ CLASS net/minecraft/class_3342 net/minecraft/structure/EndCityGenerator
 			ARG 3 pos
 			ARG 4 rotation
 			ARG 5 includeAir
+		METHOD <init> (Lnet/minecraft/class_3485;Lnet/minecraft/class_2487;)V
+			ARG 1 manager
```

Comment:
Is this `structureManager`?

Reply:
Yes, most other Structure classes simply refer to it as `manager`.